### PR TITLE
[JBPM-9844] add github actions to jbpm_system_test

### PR DESCRIPTION
**Pull request by gmunozfe**
Depends on [jbpm-playground#11](https://github.com/kiegroup/jbpm-playground/pull/11)

### CODE REVIEW

File: .github/workflows/jbpm_system_test_ci.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+name: jBPM System Test CI
+
+on:
+  schedule:
+    - cron:  '0 2 * * *'
+
+defaults:
+  run:
+    shell: bash
```
Comment (MarianMacik) [inline .github/workflows/jbpm_system_test_ci.yml:11]: FYI, according to the [docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell), `bash` is default on non-Windows platforms. But I have no issue stating that explicitly :)

**Review by MarianMacik — APPROVED** [review]
Looks good!

**Review by jstastny-cz — APPROVED** [review]
LGTM

**Comment by gmunozfe** [comment]
@mareknovotny this is already ready to be merged

**Pull request merged**